cvs diff: Diffing scp2
cvs diff: Diffing scp2/inc
cvs diff: Diffing scp2/macros
cvs diff: Diffing scp2/prj
Index: scp2/prj/build.lst
===================================================================
RCS file: /cvs/installation/scp2/prj/build.lst,v
retrieving revision 1.25
retrieving revision 1.25.36.1
diff -p -u -u -p -b -w -B -r1.25 -r1.25.36.1
--- scp2/prj/build.lst	4 Jan 2008 17:06:26 -0000	1.25
+++ scp2/prj/build.lst	12 Feb 2008 11:48:34 -0000	1.25.36.1
@@ -12,6 +12,7 @@ cp    scp2\source\graphicfilter   nmake 
 cp    scp2\source\impress         nmake    -    all    cp_impress cp_langmacros cp_langtemplates NULL
 cp    scp2\source\javafilter      nmake    -    all    cp_javafilter cp_langmacros cp_langtemplates NULL
 cp    scp2\source\kde             nmake    -    all    cp_kde cp_langmacros cp_langtemplates NULL
+cp    scp2\source\layout          nmake    -    all    cp_layout cp_langmacros NULL
 cp    scp2\source\lingu           nmake    -    all    cp_lingu cp_langmacros cp_langtemplates NULL
 cp    scp2\source\math            nmake    -    all    cp_math cp_langmacros cp_langtemplates NULL
 cp    scp2\source\ooo             nmake    -    all    cp_ooo cp_langmacros cp_langtemplates NULL
@@ -26,4 +27,4 @@ cp    scp2\source\winexplorerext  nmake 
 cp    scp2\source\ure             nmake    -    all    cp_ure cp_langmacros cp_langtemplates NULL
 cp    scp2\source\onlineupdate    nmake    -    all    cp_update cp_langmacros cp_langtemplates NULL
 cp    scp2\source\ooo_wo_ure      nmake    -    all    cp_ooo_wo_ure cp_langmacros cp_langtemplates NULL
-cp    scp2\util                   nmake    -    all    cp_util cp_activex cp_calc cp_canvas cp_crashrep cp_draw cp_gnome cp_graphicfilter cp_impress cp_javafilter cp_kde cp_lingu cp_math cp_ooo cp_python cp_quickstart cp_testtool cp_writer cp_base cp_xsltfilter cp_winexplorerext cp_ure cp_sdkoo cp_update cp_ooo_wo_ure NULL
+cp    scp2\util                   nmake    -    all    cp_util cp_activex cp_calc cp_canvas cp_crashrep cp_draw cp_gnome cp_graphicfilter cp_impress cp_javafilter cp_kde cp_layout cp_lingu cp_math cp_ooo cp_python cp_quickstart cp_testtool cp_writer cp_base cp_xsltfilter cp_winexplorerext cp_ure cp_sdkoo cp_update cp_ooo_wo_ure NULL
cvs diff: Diffing scp2/source
cvs diff: Diffing scp2/source/activex
cvs diff: Diffing scp2/source/base
cvs diff: Diffing scp2/source/calc
cvs diff: Diffing scp2/source/canvas
cvs diff: Diffing scp2/source/crashrep
cvs diff: Diffing scp2/source/draw
cvs diff: Diffing scp2/source/gnome
cvs diff: Diffing scp2/source/graphicfilter
cvs diff: Diffing scp2/source/impress
cvs diff: Diffing scp2/source/javafilter
cvs diff: Diffing scp2/source/kde
cvs diff: Diffing scp2/source/layout
Index: scp2/source/layout/layout.scp
===================================================================
RCS file: scp2/source/layout/layout.scp
diff -N scp2/source/layout/layout.scp
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ scp2/source/layout/layout.scp	12 Feb 2008 11:50:07 -0000	1.1.2.2
@@ -0,0 +1,117 @@
+/*************************************************************************
+ *
+ *  OpenOffice.org - a multi-platform office productivity suite
+ *
+ *  $RCSfile$
+ *
+ *  $Revision$
+ *
+ *  last change: $Author$ $Date$
+ *
+ *  The Contents of this file are made available subject to
+ *  the terms of GNU Lesser General Public License Version 2.1.
+ *
+ *
+ *    GNU Lesser General Public License Version 2.1
+ *    =============================================
+ *    Copyright 2005 by Sun Microsystems, Inc.
+ *    901 San Antonio Road, Palo Alto, CA 94303, USA
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License version 2.1, as published by the Free Software Foundation.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Lesser General Public
+ *    License along with this library; if not, write to the Free Software
+ *    Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ *    MA  02111-1307  USA
+ *
+************************************************************************/
+
+#include "macros.inc"
+
+File gid_File_Lib_Layout
+    TXT_FILE_BODY;
+    Styles = (PACKED,UNO_COMPONENT);
+    RegistryID = gid_Starregistry_Services_Rdb;
+    Dir = gid_Dir_Program;
+  #ifdef UNX
+    Name = STRING(CONCAT4(libtklayout,OFFICEUPD,DLLPOSTFIX,UNXSUFFIX));
+  #else
+    Name = STRING(CONCAT4(tklayout,OFFICEUPD,DLLPOSTFIX,.dll));
+  #endif
+End
+
+Directory gid_Dir_Share_Layout
+    ParentID = gid_Dir_Share;
+    DosName = "layout";
+End
+
+File gid_File_Xml_Wordcount
+   TXT_FILE_BODY;
+   Dir = gid_Dir_Share_Layout;
+   Name = "wordcount.xml";
+   Styles = (PACKED);
+End
+
+File gid_File_Xml_Zoom
+   TXT_FILE_BODY;
+   Dir = gid_Dir_Share_Layout;
+   Name = "zoom.xml";
+   Styles = (PACKED);
+End
+
+Directory gid_Dir_Share_Layout_De
+    ParentID = gid_Dir_Share_Layout;
+    DosName = "de";
+End
+
+File gid_File_Xml_Wordcount_De
+   TXT_FILE_BODY;
+   Dir = gid_Dir_Share_Layout_De;
+   Name = "de/wordcount.xml";
+   Styles = (PACKED);
+End
+
+File gid_File_Xml_Zoom_De
+   TXT_FILE_BODY;
+   Dir = gid_Dir_Share_Layout_De;
+   Name = "de/zoom.xml";
+   Styles = (PACKED);
+End
+
+Directory gid_Dir_Share_Layout_Nl
+    ParentID = gid_Dir_Share_Layout;
+    DosName = "nl";
+End
+
+File gid_File_Xml_Wordcount_Nl
+   TXT_FILE_BODY;
+   Dir = gid_Dir_Share_Layout_Nl;
+   Name = "nl/wordcount.xml";
+   Styles = (PACKED);
+End
+
+File gid_File_Xml_Zoom_Nl
+   TXT_FILE_BODY;
+   Dir = gid_Dir_Share_Layout_Nl;
+   Name = "nl/zoom.xml";
+   Styles = (PACKED);
+End
+
+//  Do we need this?
+//  Must MODULE_OPTIONAL_LAYOUT be registered somewhere?
+//
+// Module gid_Module_Layout
+//    MOD_NAME_DESC (MODULE_OPTIONAL_LAYOUT);
+//    ParentID = gid_Module_Optional;
+//    Files = (gid_File_Lib_Layout,gid_File_Xml_Wordcount,gid_File_Xml_Zoom);
+//    Minimal = NO;
+//    Default = YES;
+//    Styles = ();
+// End
Index: scp2/source/layout/makefile.mk
===================================================================
RCS file: scp2/source/layout/makefile.mk
diff -N scp2/source/layout/makefile.mk
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ scp2/source/layout/makefile.mk	12 Feb 2008 11:49:32 -0000	1.1.2.1
@@ -0,0 +1,58 @@
+#*************************************************************************
+#
+#   OpenOffice.org - a multi-platform office productivity suite
+#
+#   $RCSfile$
+#
+#   $Revision$
+#
+#   last change: $Author$ $Date$
+#
+#   The Contents of this file are made available subject to
+#   the terms of GNU Lesser General Public License Version 2.1.
+#
+#
+#     GNU Lesser General Public License Version 2.1
+#     =============================================
+#     Copyright 2005 by Sun Microsystems, Inc.
+#     901 San Antonio Road, Palo Alto, CA 94303, USA
+#
+#     This library is free software; you can redistribute it and/or
+#     modify it under the terms of the GNU Lesser General Public
+#     License version 2.1, as published by the Free Software Foundation.
+#
+#     This library is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#     Lesser General Public License for more details.
+#
+#     You should have received a copy of the GNU Lesser General Public
+#     License along with this library; if not, write to the Free Software
+#     Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+#     MA  02111-1307  USA
+#
+#*************************************************************************
+
+PRJ=..$/..
+
+PRJPCH=
+
+PRJNAME=scp2
+TARGET=layout
+TARGETTYPE=CUI
+
+# --- Settings -----------------------------------------------------
+
+.INCLUDE :	settings.mk
+
+SCP_PRODUCT_TYPE=osl
+
+PARFILES=
+
+.IF "$(ENABLE_LAYOUT)" == "TRUE"
+PARFILES+= layout.par
+.ENDIF # ENABLE_LAYOUT == TRUE
+
+# --- File ---------------------------------------------------------
+
+.INCLUDE :  target.mk
cvs diff: Diffing scp2/source/lingu
cvs diff: Diffing scp2/source/math
cvs diff: Diffing scp2/source/onlineupdate
cvs diff: Diffing scp2/source/ooo
cvs diff: Diffing scp2/source/ooo_wo_ure
cvs diff: Diffing scp2/source/python
cvs diff: Diffing scp2/source/quickstart
cvs diff: Diffing scp2/source/sdkoo
cvs diff: Diffing scp2/source/templates
cvs diff: Diffing scp2/source/testtool
cvs diff: Diffing scp2/source/ure
cvs diff: Diffing scp2/source/winexplorerext
cvs diff: Diffing scp2/source/writer
cvs diff: Diffing scp2/source/xsltfilter
cvs diff: Diffing scp2/util
Index: scp2/util/makefile.mk
===================================================================
RCS file: /cvs/installation/scp2/util/makefile.mk,v
retrieving revision 1.43
retrieving revision 1.43.36.1
diff -p -u -u -p -b -w -B -r1.43 -r1.43.36.1
--- scp2/util/makefile.mk	4 Jan 2008 17:13:28 -0000	1.43
+++ scp2/util/makefile.mk	12 Feb 2008 11:48:34 -0000	1.43.36.1
@@ -172,6 +172,10 @@ SCP1FILES += \
 SCP1FILES += cairocanvas.par
 .ENDIF
 
+.IF "$(ENABLE_LAYOUT)" == "TRUE"
+SCP1FILES += layout.par
+.ENDIF # ENABLE_LAYOUT == TRUE
+
 .IF "$(BUILD_SPECIAL)"!=""
 SCP1FILES += \
              module_onlineupdate.par   \
@@ -303,6 +307,10 @@ SCP2FILES += \
 SCP2FILES += \
              vclcanvas.par
 
+.IF "$(ENABLE_LAYOUT)" == "TRUE"
+SCP2FILES += layout.par
+.ENDIF # ENABLE_LAYOUT == TRUE
+
 .IF "$(BUILD_SPECIAL)"!=""
 SCP2FILES += \
              module_onlineupdate.par   \
